exercises/practice/scale-generator/.docs/instructions.md

```diff
@@ -1,37 +1,48 @@
 # Instructions
 
-Given a tonic, or starting note, and a set of intervals, generate
-the musical scale starting with the tonic and following the
-specified interval pattern.
+## Chromatic Scales
 
 Scales in Western music are based on the chromatic (12-note) scale. This
 scale can be expressed as the following group of pitches:
 
-A, A#, B, C, C#, D, D#, E, F, F#, G, G#
+> A, A#, B, C, C#, D, D#, E, F, F#, G, G#
 
 A given sharp note (indicated by a #) can also be expressed as the flat
 of the note above it (indicated by a b) so the chromatic scale can also be
 written like this:
 
-A, Bb, B, C, Db, D, Eb, E, F, Gb, G, Ab
+> A, Bb, B, C, Db, D, Eb, E, F, Gb, G, Ab
```

Comment:
Should these change to actual utf flat characters (U+266D, or maybe there's a combining superscript?) instead of using the letter b?

Reply:
🤷 Maybe open an issue (or a PR) at https://github.com/exercism/problem-specifications?